### PR TITLE
fix: use typed object params for itly.load()

### PR DESCRIPTION
**Pull request by falconandy**

### CODE REVIEW

File: packages/sdk/lib/base.ts [hunk 1]
```diff
@@ -184,20 +191,24 @@ export class Itly {
 
   /**
    * Initialize the Itly SDK. Call once when your application starts.
-   * @param context Additional context properties to add to all events.
    * @param options Configuration options to initialize the Itly SDK with.
    */
-  load(context?: Properties, options?: Options) {
+  load(options: LoadOptions = {}) {
```
Comment (justin-iteratively) [inline packages/sdk/lib/base.ts:196]: I think renaming this parameter `loadOptions` would make the following code a little easier.

E.g.
```
const {
  context,
  ...options
} = loadOptions;
```

File: packages/sdk/lib/browser.ts [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import {
   Itly as ItlyBase,
   Options,
+  LoadOptions,
```
Comment (justin-iteratively) [inline packages/sdk/lib/browser.ts:5]: Need to also export `LoadOptions` in `packages/sdk/lib/node.ts`

File: _templates/sdk/new/lib-index.ts.t [hunk 3]
```diff
@@ -29,8 +31,8 @@ export class Itly {
   }
 
   load = (
-    options: Options,
-  ) => this.itly.load(options);
+    options: LoadOptions = {},
+  ) => this.itly.load(params);
```
[the remark this answers is not in the document]
Comment (falconandy) [reply _templates/sdk/new/lib-index.ts.t]: Done

File: packages/sdk/lib/base.ts (hunk 1 above)
Comment (falconandy) [reply packages/sdk/lib/base.ts]: Done

File: packages/sdk/lib/browser.ts (hunk 2 above)
Comment (falconandy) [reply packages/sdk/lib/browser.ts:5]: Done

**Review by justin-iteratively — APPROVED** [review]

**Pull request merged**